appeals/web/src/server/lib/mappers/notification-banners.mapper.js

```diff
@@ -73,6 +73,12 @@ export const notificationBannerDefinitions = {
 	readyForDecision: {
 		pages: ['appealDetails']
 	},
+	readyForValidation: {
+		pages: ['appealDetails']
```

Comment:
I think you may need `persist: true` in these new definitions, in order to keep the banners from disappearing after they are displayed once

Reply:
these banners are persisting and I've possibly found a bug on this. Will create a separate ticket for it.